_rules/html-xml-lang-match-5b7ae0.md

```diff
@@ -72,15 +72,7 @@ Since most assistive technologies will consistently use `lang` over `xml:lang` w
 
 #### Failed Example 2
 
-`html` element with non-matching primary language subtags for `lang` and `xml:lang`.
-
-```html
-<html lang="yue" xml:lang="cmn"></html>
-```
-
-#### Failed Example 3
-
-`html` element with non-matching primary and extended language subtags for `lang` and `xml:lang`.
+`html` element with non-matching extended language subtags for `lang` and `xml:lang`. While the primary language subtag is Chinese for both the `lang` and `xml:lang` attribute, the extended language subtag is Cantonese for the `lang` attribute and Mandarin for the `xml:lang` attribute.
```

Comment:
@Jym77 As I don't know anything at all about the relationship between the different dialects of Chinese, this is about as good an explanation I could come up with for why the extended language subtags matter. Do you have any suggestions for improvements?

Reply:
It's not just extended language subtags - the script also matters - you can write Chinese Mandarin in Han Traditional, Han Simplified script (which uses simpler character shapes than traditional script) or Pinyin script (uses Latin characters to phonetically spell words - very similar to Rōmaji in Japanese)

The word 'dragon' has 3 different forms in different scripts:

- Dragon in Han Simplified is 龙 (zh-cmn-Hans or cmn-Hans)
- Dragon in Han Traditional is 龍 (zh-cmn-Hant or cmn-Hant)
- Dragon in Pinyin is lóng (zh-Latn-pinyin)

See https://sites.psu.edu/symbolcodes/languages/asia/chinese/
and https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

Reply:
Is there any effect on how AT announces these though? For example, would something like the following be announced correctly?

```html
<p lang="zh-cmn-Hant">龙</p>
```

Reply:
Han Simplified and Han Traditional characters have different shapes, which means they're represented by different Unicode code points - that might be enough to pronounce them correctly. It might be more complicated than this because some Han characters are also used in Japanese (where they're called Kanji and pronounced differently).

For Pinyin/Rōmaji then script might matter to get the latin letters pronounced correctly. In Japanese Rōmaji latin letters shouldn't be pronounced exactly like English or any other European language because Japanese has a more limited set of sounds (phonemes) compared to European languages.

More investigation needed...

Reply:
There seem to exists a `zh-Latn-wadegiles` tag: 
https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry (and  search for "wadegiles" in the page). And I am sure that while this uses latin characters, this should be pronounced pretty differently than `zh-Latn-pinyin` (http://www.eastasianlib.org/ctp/RomTable/Chipinyintowade.pdf)

So script subtags (and their extension) might be important…

(it seems that `ja-Latn` only has a `hepburn` (and `heploc`) subtag, but nothing for Nihon-shiki or Kunrei romanisation (https://www.japanesewithanime.com/2017/12/romaji-systems-hepburn-nihon-kunrei.html))


The difference in pronunciation between Han characters in Chinese and kanjis in Japanese should be taken into account by the primary language subtag (_i.e._ `<p lang="zh">日</p>` and `<p lang="ja">日</p>` should be enough to have a difference in pronunciation).

Reply:
Resolving this, let's move the discussion to a separate issue or pull request.